frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -313,17 +336,6 @@ local note_mark = {
     {_("Side mark"), "sidemark"},
 }
 
-local long_press_action = {
-    {_("Ask with popup dialog"), "ask"},
-    {_("Do nothing"), "nothing"},
```

Comment:
Why not just move this one where you put the 2 new ones (and invert the values if it makes more sense, `{"ask", _("Ask with popup dialog") }` ?